scripts/gpu_ram_info.sh

```diff
@@ -42,14 +42,14 @@ get_gpu()
   gpu_vram_percent=$(get_tmux_option "@dracula-gpu-vram-percent" false)
   if [[ "$gpu" == NVIDIA ]]; then
     if $gpu_vram_percent; then
-      usage=$(nvidia-smi --query-gpu=memory.used,memory.total --format=csv,noheader,nounits | awk '{ used += $0; total +=$2 } END { printf("%d%%\n", used / total * 100 ) }')
+      usage=$(nvidia-smi --query-gpu=memory.used,memory.total --format=csv,noheader,nounits | awk -F ', *' '{ printf("|%d%%", $0 / $2 * 100) }' && echo "|")
     normalize_percent_len $usage
```

Comment:
This too should be an echo because of this:

<img width="379" height="20" alt="Image" src="https://github.com/user-attachments/assets/3e990039-de82-4184-9174-a06164ba2524" />

```suggestion
    echo $usage
```